### PR TITLE
Ownft project

### PR DESCRIPTION
**Pull request by sha256rma**
Adding our project to the IPFS Community Showcase for https://github.com/protocol/grants/blob/main/nextstep.md

### CODE REVIEW

File: projects/Ownft.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+- https://github.com/denny999222
+
+Amit
+- front-end developer
+- Information Technology major student
+- https://github.com/akumar1999
+
+
+## How the community can engage
+* Email: ksharma@dype.market
```
Comment (autonome) [inline projects/Ownft.md:96]: Please see the requirements - you need to make a discussion thread in this repo's Discussions section and link it here.

File: projects/Ownft.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+
+Our hardware + software stack is currently under development so we do not have any users
+
+## Github repo
+<!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+
+https://github.com/sha256rma/ownft_v2
```
Comment (autonome) [inline projects/Ownft.md:53]: Needs MIT license to qualify for the grant.

**Review by autonome — CHANGES_REQUESTED** [review]

File: projects/Ownft.md (hunk 2 above)
Comment (sha256rma) [reply projects/Ownft.md:53]: Added MIT license to project. Please check again.

**Review by sha256rma — COMMENTED** [review]
Discussion thread of Ownft: https://github.com/ipfs/community/discussions/550

File: projects/Ownft.md (hunk 1 above)
Comment (sha256rma) [reply projects/Ownft.md:96]: https://github.com/ipfs/community/discussions/550

**Review by sha256rma — COMMENTED** [review]
added discussion link: https://github.com/ipfs/community/discussions/550

**Pull request merged**